plugins/withAndroidDayNightThemePlugin.js

```diff
@@ -0,0 +1,27 @@
+// Based on https://github.com/expo/expo/pull/33957
```

Comment:
Add proper support for dark mode in Expo (see https://github.com/expo/expo/pull/33957)